SocialFilter/sections/popups.txt

```diff
@@ -9,9 +9,9 @@
 ||my.hellobar.com^$third-party
 !
 !+ PLATFORM(android, ios, ext_android_cb)
```

Comment:
Android does not need the rules when you have cookie rule (applies to Annoyances too).